.readthedocs.yaml

```diff
@@ -0,0 +1,17 @@
+# SPDX-FileCopyrightText: 2017 Free Software Foundation Europe e.V. <https://fsfe.org>
+#
+# SPDX-License-Identifier: CC0-1.0
+
+version: 2
+
+build:
+  os: "ubuntu-22.04"
+  tools:
+    python: "3.10"
```

Comment:
This should probably mirror our CI configuration, which uses 20.04 and Python 3.6, with a comment stating that these two should be identical.